extide/gradle/netbeans-gradle-tooling/src/main/java/org/netbeans/modules/gradle/tooling/NbProjectInfoBuilder.java

```diff
@@ -445,10 +445,14 @@ private void detectDependencies(NbProjectInfoModel model) {
                                 // do not bother with components that only select a variant, which is itself a component
                                 // TODO: represent as a special component type so the IDE shows it, but the IDE knows it is an abstract
                                 // intermediate with no artifact(s).
-                                if (rdr.getResolvedVariant() == null ||
-                                    !rdr.getResolvedVariant().getExternalVariant().isPresent()) {
+                                if (rdr.getResolvedVariant() == null) {
                                     componentIds.add(rdr.getSelected().getId().toString());
                                 }
```

Comment:
I'd put the `sinceGradle` into `else` branch, so that executes just for `resolvedVariant != null` case.

Reply:
True! Thanks! Fixed.